aws/aws.go

```diff
@@ -190,6 +190,25 @@ func encodeFilterPolicy(filter map[string]string) (*string, error) {
 	return aws.String(string(bytes)), err
 }
 
+// decodeFilterPolicy converts the given string from a sns.GetSubscriptionAttributesOutput.Attributes
+// value into a map to meet the subscriber interface
+func decodeFilterPolicy(filterPolicy *string) (map[string]string, error) {
+	decoded := make(map[string]string)
+	if filterPolicy != nil {
```

Comment:
nit: instead of this if just do:
```
if filterPolicy == nil {
  return decoded, nil
}
... etc ...
```

So the whole method doesn't have an extra indent level.